src/menu/menu_functions.rs

```diff
@@ -88,7 +88,7 @@ pub fn parse_selection_char(buffer: &str, marker: char) -> ParseResult {
                 }
                 Some(&x) if x.is_ascii_digit() || x == '-' => {
                     let mut count: usize = 0;
-                    let mut size: usize = 1;
+                    let mut size: usize = marker.len_utf8();
```

Comment:
Correct me if I am wrong but doesn't size refer to the digit or `-` sign which is guaranteed to be single byte ascii in this case.

Reply:
Yeah, marker.len_utf8() would always be 1 here.   I will change it back :)

Reply:
This is needed to make this test succeed, since the marker is a multi byte char.  
```
fn handles_multi_byte_char_as_marker_and_number() {
        let buffer = "Testは4!";
        let parse_result = parse_selection_char(buffer, 'は');

        assert_eq!(parse_result.remainder, "Test");
        assert_eq!(parse_result.index, Some(4));
        assert_eq!(parse_result.marker, Some("は4"));
    }
```

Reply:
The size is only used to pull out the marker value.  
```
marker: Some(&buffer[index..index + size]),
```
Usually it would just be the one character (marker.len_utf8()) but it will expand to pull in the number or - as well.